src/Client.php

```diff
@@ -202,6 +202,14 @@ public function flush(?int $timeout = null): PromiseInterface
         return $this->transport->close($timeout);
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getStacktraceBuilder(): StacktraceBuilder
```

Comment:
We should make this method part of the interface. To avoid a BC, it should be done with the `@method` annotation

Reply:
Does this do it? I don't have a PhpStorm etc. on this laptop to test

Reply:
Yes it does 👍